### PR TITLE
External links open in new tabs

### PR DESCRIPTION
**Pull request by malloryfreeberg**
## Description

- Updated all external links to open in a new tab using embedded html
- Updated all internal links to be relative and open in same window
- Fixed typos & improved unclear text
- Updated any links with text "here" to text describing what the linked material is (better UX, consistent with rest of website)

## Motivation and Context

Currently, links to external materials/sites are opened in the same tab that is currently being used. If someone wants to go back to the FEGA pages, they have to navigate back from where they are. A better user experience would be that external links open in a new tab in the same window, so that a user can easily switch back to the FEGA pages without losing access to the external materials they just opened. Internal links to other pages in the FEGA website can stay in the same tab, since it doesn't navigate someone away from the FEGA website. This PR addressed Issue #26.

## How has this been tested?

Changes have been tested in my fork by enabling GitHub pages to build a local copy of the website. All website links were then checked (using Mac & Google Chrome) that:
- External links opened in a new tab
- Internal links opened in the same tab

**NB** Links _won't_ open correctly in GitHub markdown; GitHub pages must be built to test.

## What should reviewers focus on?

- External links open in a new tab and internal links open in same window, optionally in another OS/browser
- Updated text is at least as or more clear than before

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New content (non-breaking change which adds new content)
- [x] Modified content (non-breaking change which modifies existing content)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [x] I have read the CONTRIBUTING.md guide.
- [x] My changes follow the code style of this project.
- [x] I have updated supporting documentation, if required.
- [ ] I have added my details to the CONTRIBUTORS.yaml file.

### CODE REVIEW

File: docs/topics/data-metadata-management/index.md [hunk 1]
```diff
@@ -19,52 +19,54 @@ Welcome! If you are involved in data or metadata standards aspects of establishi
...
+| refget | Enables access to reference sequences using an identifier derived from the sequence itself. | v1.2.6 | N/A | <a href="http://samtools.github.io/hts-specs/refget.html" target="_blank">Specification</a>, Documentation, Endpoint | <a href="https://doi.org/10.1093/bioinformatics/btab524" target="_blank">DOI</a> |
+| Researcher IDs (Passports, Visas) | Specifies the collection of researchers that may access a dataset at any given time, and the credentials they must supply. | v1.0.1 | v1.0.1 | <a href="https://github.com/ga4gh-duri/ga4gh-duri.github.io/blob/master/researcher_ids/ga4gh_passport_v1.md" target="_blank">Specification</a>, <a href="https://docs.google.com/document/d/1FTzUYAfV5d2a0zoDkbY9Iy_L5NbSAnHeWnmY2NIrY8M/edit" target="_blank">Documentation</a>, <a href="https://ega.ebi.ac.uk:8443/ega-permissions/swagger-ui/index.html" target="_blank">Endpoint</a> | <a href="https://doi.org/10.1016/j.xgen.2021.100030" target="_blank">DOI</a> |
```
Comment (M-casado) [inline docs/topics/data-metadata-management/index.md:49]: Out of curiosity, why have Visas in uppercase? I've seen the DOI where they put the "passport" specification with uppercase, but not visas. And wouldn't it be "Passports and visas"?

**Review by M-casado — APPROVED** [review]
LGTM. Checked a few links of each page and they work fine (in [Mallory's forked GitHub pages](https://malloryfreeberg.github.io/FEGA-onboarding/)). 
I have not checked the full set of documents, though, searching for external links that escaped this PR.

File: docs/topics/data-metadata-management/index.md (hunk 1 above)
Comment (M-casado) [reply docs/topics/data-metadata-management/index.md:49]: I just wanted to highlight line 49, but git highlighted all these. Sorry about that 😃

Comment (malloryfreeberg) [reply docs/topics/data-metadata-management/index.md]: Changed to "visas"

**Pull request merged**